dojo_plugin/api/v1/score.py

```diff
@@ -0,0 +1,48 @@
+from flask import request
+from flask_restx import Namespace, Resource
+from CTFd.cache import cache
+from CTFd.models import Users
+
+from ...models import Dojos
+
+score_namespace = Namespace("score")
+
+def get_user_score(username):
+    # Alternative: hardcode max count and challenge id's for speed
+    # POSSIBLE BUG HERE - topics and courses dojos might result in doubling
+    # but since solves carry over, it should not be an issue
+    # Maybe have a proper scoring system where babyarch has more weight than babysuid
```

Comment:
I agree with this concern, but I also agree that we can just go ahead and keep it simple for now.